business-central-parent/business-monitoring-webapp/pom.xml

```diff
@@ -550,10 +550,6 @@
     </dependency>
 
     <!-- Errai Core -->
-    <dependency>
-      <groupId>org.jboss.errai</groupId>
-      <artifactId>errai-bus</artifactId>
```

Comment:
The other exclusions are ok to me, but this is quite a core one... sure we can remove this?